spinoso-time/src/time/tzrs/math.rs

```diff
@@ -15,7 +15,7 @@ impl Time {
     /// # Examples
     /// ```
     /// use spinoso_time::tzrs::Time;
-    /// let now = Time::local(2010, 3, 30, 5, 43, 25, 123456789);
+    /// let now = Time::local(2010, 3, 30, 5, 43, 25, 123456789).unwrap();
```

Comment:
usually I add some boilerplate to the doc test to allow the example to model good behavior with `?`, like this:

https://github.com/b-n/artichoke/blob/a3c85a2d374c72ab77490b171e23529c06dfeba3/spinoso-array/src/array/tinyvec/mod.rs#L965-L974

this one is with an `Option`, but the setup and `example().unwrap()` (which makes sure the test gets run!) is the same.